### ISSUE TITLE
Text highlights in High Contrast theme

### ISSUE THREAD

**Issue by ethan-fang-MS** [microsoft/devhome#1752]
### Windows build number

10.0.25977.0

### Steps to reproduce the bug

Turn on high contrast (Night Sky) and open Dev Home

### Expected result

Rather than having a black highlight, the text itself should be black with no highlight.

This is an issue for text in:

- The navigation pane
- Banners
- Buttons

### Actual result

Text is white with a black highlight.

![image](https://github.com/microsoft/devhome/assets/117125208/00ea5bac-19c0-4620-ba79-353b30e1c500)

**Comment by krschau** [comment]
@ethan-fang-MS I thought the banner was correct, it's the buttons and nav pane that were the issue.
